make-uniform/transitland_feed/standard_route_name_db.R

```diff
@@ -19,3 +19,173 @@ standard_routes <- data.frame()
+  
+  mutate(base_name = ifelse(str_detect(entry, "^SamTrans "), 
+                            str_replace(entry, "^SamTrans ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^SamTrans "), "SamTrans", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^San Francisco Bay Ferry "), 
+                            str_replace(entry, "^San Francisco Bay Ferry ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^San Francisco Bay Ferry "), "San Francisco Bay Ferry", operator)) %>%
+  
+  mutate(operator = ifelse(str_detect(entry, "SFSU"), "SFSU", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^Stanford "), 
+                            str_replace(entry, "^Stanford ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^Stanford "), "Stanford", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^SolTrans "), 
+                            str_replace(entry, "^SolTrans ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^SolTrans "), "SolTrans", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^Tri Delta "), 
+                            str_replace(entry, "^Tri Delta ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^Tri Delta "), "Tri Delta", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^UCSF "), 
+                            str_replace(entry, "^UCSF ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^UCSF "), "UCSF", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^Union City "), 
+                            str_replace(entry, "^Union City ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^Union City "), "Union City", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^VINE 29 "), 
+                            str_replace(entry, "^VINE 29 ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^VINE "), "Napa Vine", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^VTA "), 
+                            str_replace(entry, "^VTA ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^VTA "), "VTA", operator)) %>%
+  
+  mutate(base_name = ifelse(str_detect(entry, "^WestCAT "), 
+                            str_replace(entry, "^WestCAT ", ""),
+                            base_name)) %>%
+  mutate(operator = ifelse(str_detect(entry, "^WestCAT "), "WestCAT", operator)) %>%
+  
+  mutate(operator = ifelse(operator == "", "BAD REFERENCE", operator))
+
+unique_routes <- unique_routes %>%
+  mutate(survey = "SFMTA") %>%
```

Comment:
`survey` should be consistent with whatever we use in the `read_operator` method in `Build Survey Standard.Rmd`, e.g., 

```
muni_df <- read_operator('SF Muni',
                         2017,
                         'local bus',
                         f_muni_survey_path,
                         dictionary_all)
```
Can we take it from an entry in the database?

Reply:
The route names in the survey all reference "MUNI". I'll just change survey to 'SF Muni' and update the operators to be consistent as well.